scripts/generate-schemas.py

```diff
@@ -267,14 +284,40 @@ def build_question_properties(question):
     return question_data
 
 
+def build_any_of(any_of, fields):
+    return {
+        'required': [field for field in sorted(fields)],
+        'title': any_of
+    }
+
+
 def build_schema_properties(schema, questions):
+    any_ofs = {}
```

Comment:
Dictionary converts quite easily to a sorted list is all.  Otherwise, the order isn't guaranteed.